### ISSUE TITLE
"Conversion could overflow variable" rule does not catch INTEGER(RecId).

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#925]
Since RECID is INT64, this can cause overflow errors and currently is not caught by the rule.

```
DEFINE VARIABLE lcRecId AS RECID NO-UNDO.
DEFINE VARIABLE liInt AS INTEGER NO-UNDO.
liInt = INTEGER(lcRecId).
```

**Comment by gquerret** [comment]
Fixed
